unstructured/ingest/connector/confluence.py

```diff
@@ -22,6 +24,11 @@
     from atlassian import Confluence
 
 
+@dataclass
+class ConfluenceAccessConfig(AccessConfig):
+    api_token: str = enhanced_field(sensitive=True)
```

Comment:
should this now be access_config, or have the override name?

Reply:
Nevermind. This is now in the access_config as ConfluenceAccessConfig. Good stuff.